### ISSUE TITLE
Database update doesn't check for removed albums

### ISSUE THREAD

**Issue by WatermelonDriveway** [arybczak/ncmpcpp#202]
If I delete an album from my music directory and press 'u' to update my database, it doesn't notice that I have removed the album.

If I use the browser window '2' and navigate to the specific album I deleted and then run the update command, it will remove it.

**Comment by arybczak** [comment]
When update is issued, all ncmpcpp does is sending the update command to MPD, so I can't help you here.

**Comment by arybczak** [comment]
> If I delete an album from my music directory and press 'u' to update my database, it doesn't notice that I have removed the album.

Where are you pressing 'u'?

**Comment by WatermelonDriveway** [comment]
I am usually in the playlist or media library windows, so either one of those

**Comment by rewozz** [comment]
Yes this is easy to reproduce, add a folder with a music file then delete that folder in the file view (2). Repeat the process and you get this:

![screen shot 2017-05-03 at 15 16 23](https://cloud.githubusercontent.com/assets/4593500/25662126/a216052a-3013-11e7-87eb-11086a024209.png)

Thanks for the temporary workaround @WatermelonDriveway

**Comment by bbsixzz** [comment]
Just want to add there is no workaround for single files, only folders.
